doc/middleware-it-registratore-telematico/operation-modes/scu/customserver.md

```diff
@@ -0,0 +1,34 @@
+---
+slug: /poscreators/middleware-doc/italy/scu/customserver
+title: Custom-Server
+---
+
+# Custom Server
+
+## Signature Creation Unit
+
+### Support
+
+**Alpha from version:** 1.3.45
+
+**Stable from version:** TBD
+
+The _fiskaltrust.Middleware.SCU.IT.CustomRTServer_ package connects the middleware with a Custom-Server.
+
+### Parameters
+
+| Name | Description | Optional |
+| ---- | ------------ |--------- |
+| DeviceUrl| The URL or IP address of the RT Printer or Server | mandatory |
```

Comment:
Paramters are not correct IMO please double check with the config

Reply:
done